analyses/snv-callers/scripts/01-calculate_vaf_tmb.R

```diff
@@ -249,9 +276,14 @@ if (file.exists(vaf_file) && !opt$overwrite) {
 
   message(paste(nrow(vaf_df), "mutations left after filter and merge"))
 
-  # Write this to a TSV file
-  vaf_df %>% readr::write_tsv(vaf_file)
-
+  # Write the vaf file
+  if (opt$file_format == "rds") {
```

Comment:
If you flip the order of these, that would be more consistent with RDS being the default, i.e., save a TSV if you asked for a TSV, otherwise save an RDS. Also seems to be what you do in the other file.

Reply:
Will do.